features-json/css-image-set.json

```diff
@@ -429,8 +429,9 @@
   "notes":"",
   "notes_by_num":{
     "1":"Has very limited support. Only `url()` is accepted as the image and only `x` is accepted as a resolution.",
-    "2":"Does not support generated images (example: `linear-gradient()`).",
-    "3":"Does not support the `type()` function. "
+    "2":"Does not support generated images in cursor and conten properties (example: `linear-gradient()`) [Bug 1107646](https://bugzilla.mozilla.org/show_bug.cgi?id=1107646).",
```

Comment:
This is just a modification of the original content, because except cursor, other properties support linear-gradient(), I did not find a separately recorded bug url, so I can consider removing it here.

Reply:
Change to https://bugzilla.mozilla.org/show_bug.cgi?id=1696314

Reply:
Oh, now I understand! I completely missed that generated images are supported in `background-image`. I have a few corrections, otherwise this seems good ⬇️